cmd/root.go

```diff
@@ -201,3 +204,18 @@ func initConfig() {
 		fmt.Println("Using config file:", viper.ConfigFileUsed())
 	}
 }
+
+// Checks the server to ensure it has ephemeral containers.
+func hasEphemeralContainersResource(client *clientset.Clientset) bool {
```

Comment:
Is this something that maybe should be part of a helpers file? Maybe in `pkg/sidecarterminator`?

Reply:
Hmm I see. Yeah I suppose since it is only used in a single instance it does allow readers to get an understanding immediately about what is involved in checking for that compatibility without having to search the rest of the codebase.

Good thinking here 😃